core/vm/contracts.go

```diff
@@ -93,7 +93,8 @@ func ecrecoverFunc(in []byte) []byte {
 	vbig := common.Bytes2Big(in[32:64])
 	v := byte(vbig.Uint64())
 
-	if !crypto.ValidateSignatureValues(v, r, s) {
+	// tighter sig s values in homestead only apply to tx sigs
```

Comment:
@obscuren Now I saw the skype convo about it - I'll make a comment on the EIP so we can summarise the discussion in public and amend EIP if needed.